lldb/tools/lldb-dap/src-ts/uri-launch-handler.ts

```diff
@@ -0,0 +1,37 @@
+import * as vscode from "vscode";
+
+export class LaunchUriHandler implements vscode.UriHandler {
+    async handleUri(uri: vscode.Uri) {
+        try {
+            const params = new URLSearchParams(uri.query);
+            if (uri.path == '/launch/config') {
```

Comment:
We can support both launch and attach with a minor change.
Just make the expected url to be vscode://llvm-vs-code-extensions.lldb-dap/start?request=attach|launch&...
I think this leads to a better mapping between debug configs and url triggers.